openfisca_core/tools/test_runner.py

```diff
@@ -31,7 +31,22 @@ def import_yaml():
     return yaml, Loader
 
 
-TEST_KEYWORDS = {'absolute_error_margin', 'description', 'extensions', 'ignore_variables', 'input', 'keywords', 'max_spiral_loops', 'name', 'only_variables', 'output', 'period', 'reforms', 'relative_error_margin'}
+TEST_KEYWORDS = {
+    'absolute_error_margin',
+    'description',
+    'extensions',
+    'ignore_variables',
+    'input',
+    'keywords',
+    'max_spiral_loops',
+    'name',
+    'neutralized_variables',
```

Comment:
Could you add an example in the [doc](https://openfisca.org/doc/coding-the-legislation/writing_yaml_tests.html#syntax)?

Reply:
Done in https://github.com/openfisca/openfisca-doc/pull/252